.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         UPLOAD_PACKAGES: 'True'
   timeoutInMinutes: 360
   variables:
-    CONDA_BLD_PATH: D:\\bld\\
+    CONDA_BLD_PATH: C:\bld\
```

Comment:
Probably won't work. I thought that the `D:` use case was deep nested into subdirs :-/

Reply:
Is there a way to pin the version of setuptools used by the build process? My theory is that downgrading to 62.x might make the problem go away. Pinning here wouldn't be a real fix, of course, but it would be nice to be able to test the theory.

Reply:
We can pin but yeah, ideally we should fix this for good. Trying that now...

Reply:
The pin I was thinking of would apply to the builder's environment — i.e. the Python process invoking conda-build, not the host environment of the package. I think maybe something changed in setuptools that caused conda-build to do the vcvars setup twice, leading to the issue.

Reply:
Sorry, did not pay enough attention to the issue. We can pin but that would be a bit disruptive. I'll hack the CIs here tomorrow to test if this fixes it.

Reply:
@ocefpaf See below, no need — I think I was on the wrong trail anyway.